src/ui/WestRegionTrackerItems.js

```diff
@@ -57,6 +61,11 @@ WestRegionTrackerItems = function(refs) {
             reader: {
                 type: 'json',
                 root: 'programs',
+                getResponseData(response) {
+                    const data = JSON.parse(response.responseText)
```

Comment:
This will throw a runtime error if the JSON parse fails, is that what we want here?

Reply:
The [default implementation of `Ext.data.reader.Json.getResponseData`](https://docs.sencha.com/extjs/4.0.7/source/Json2.html#Ext-data-reader-Json-method-getResponseData) also raises a runtime error if JSON parsing fails, with the only difference being that it prints out a more useful stack trace.

I agree that it would be better if any errors that occur in loading event programs would be flagged to the user and if `d2-analysis` would attempt to refetch but such changes would probably require extensive refactoring.